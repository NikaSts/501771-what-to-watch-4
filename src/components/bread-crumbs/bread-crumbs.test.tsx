import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {Router} from 'react-router-dom';
import history from '../../history';
import BreadCrumbs from './bread-crumbs';

const id = 1;
const title = ``;

it(`Overview should render correctly`, () => {
  const tree = renderer
    .create(
        <Router history={history}>
          <BreadCrumbs
            id={id}
            title={title}
          />
        </Router>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
