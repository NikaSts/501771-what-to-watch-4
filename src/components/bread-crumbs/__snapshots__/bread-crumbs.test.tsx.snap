// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview should render correctly 1`] = `
<nav
  className="breadcrumbs"
>
  <ul
    className="breadcrumbs__list"
  >
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
        href="/films/1"
        onClick={[Function]}
      >
        
      </a>
    </li>
    <li
      className="breadcrumbs__item"
    >
      <a
        className="breadcrumbs__link"
      >
        Add review
      </a>
    </li>
  </ul>
</nav>
`;
